Sources/SwiftGenKit/Parsers/StoryboardParser/Storyboard+XML.swift

```diff
@@ -0,0 +1,107 @@
+//
+// SwiftGenKit
+// Copyright (c) 2017 SwiftGen
+// MIT Licence
+//
+
+import Foundation
+import Kanna
+
+private enum XML {
+  enum Scene {
+    static let initialVCXPath = "/*/@initialViewController"
+    static let targetRuntimeXPath = "/*/@targetRuntime"
+    static func initialSceneXPath(identifier: String) -> String {
+      return "/document/scenes/scene/objects/*[@sceneMemberID=\"viewController\" and @id=\"\(identifier)\"]"
+    }
+    static let sceneXPath = "/document/scenes/scene/objects/*[@sceneMemberID=\"viewController\"]"
+    static let sceneSegueXPath = "//connections/segue"
```

Comment:
For my understanding, the StoryboardSegue-enum that is created inside the extensions of each custom Scene-class should only contain the segues that are started from that specific scene.
If this is the case, the xpath has to be "./connections/segue"

Reply:
The reason for the `//` in `sceneSegueXPath` is to be able to match segues started from (for example) container views (-> embed segues). `sceneSegueXPath` is only used to match children of a scene object:
https://github.com/SwiftGen/SwiftGen/pull/364/files#diff-aab03c1cbf39dc54bf61cfae345dfc87R43